src/components/TableStyle.css

```diff
@@ -0,0 +1,182 @@
+div {
+     margin:  auto; 
+    margin-top: 5px;
+    max-width: 1200px;
+    font-family: 'Arial', sans-serif;
+    background-color: #fff;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .heading {
+    text-align: center;
+    color: #333;
+    font-size: 28px;
+    margin-bottom: 20px;
+  }
+  
+  .filter-container {
+    width: 80%;
+    max-width: 800px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    /* margin-bottom: 20px; */
+  }
+  
+  label {
+    font-size: 16px;
+    margin-right: 10px;
+    color: #555;
+  }
+  
+  input[type="date"] {
+    padding: 10px;
+    font-size: 14px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    width: 150px;
+    margin-right: 10px;
+    max-width: 200px;
+  }
+  
+  .btn {
+    background-color: #28a745;
+    color: white;
+    border: none;
+    padding: 10px 20px;
+    border-radius: 4px;
+    cursor: pointer;
+    font-size: 14px;
+    transition: background-color 0.3s ease;
+  }
+  
+  .btn:hover {
+    background-color: #218838;
+  }
+  
+  .btn:focus {
+    outline: none;
+  }
+  
+  /* Reset Button */
+ .back-btn{
+  background-color: #28a745;
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 4px;
+  cursor: pointer;
+  font-size: 14px;
+  transition: background-color 0.3s ease;
+  }
+  
+  .back-btn:hover {
+    background-color: #c82333;
+  }
+
+  .pagination {
+    display: flex;
+    flex-direction: row  ;
+  }
+
+  .active-page {
+    color: gray;
+  }
+  
+ 
+
+  table {
+    width: 100%;
+    border-collapse: collapse;
+    /* margin-top: 20px; */
+    border-radius: 8px;
+    overflow: hidden;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+  }
+  
+  thead {
+    background-color: #f1f1f1;
+  }
+  
+  th, td {
+    padding: 12px;
+    text-align: left;
+    font-size: 14px;
+    color: #333;
+    border-bottom: 1px solid #ddd;
+  }
+  
+  th {
+    background-color: #007bff;
+    color: white;
+  }
+  
+  tbody tr:nth-child(even) {
+    background-color: #f9f9f9;
+  }
+  
+  tbody tr:hover {
+    background-color: #f1f1f1;
+  }
+  
+  tbody tr:last-child td {
+    border-bottom: none;
+  }
+  
+  p {
+    font-size: 16px;
+    color: #555;
+    /* margin: 10px 0; */
+    text-align: center;
+  }
+  
+  p.total {
+    font-weight: bold;
+    color: #007bff;
+  }
+  
+  @media (max-width: 768px) {
```

Comment:
Genrally try to aviod duplicate css. For example I have refactored the below
 Refactored table font-size for media queries 
@media (max-width: 768px) {
  table, th, td {
    font-size: 12px;
    padding: 10px;
  }
}

@media (max-width: 480px) {
  table, th, td {
    font-size: 10px;
    padding: 8px;
  }
}